src/aboutDialog.ui

```diff
@@ -8,7 +8,7 @@
     <property name="type_hint">dialog</property>
     <property name="program_name">Flatseal</property>
     <property name="version">1.2.2</property>
-    <property name="copyright" translatable="yes">Copyright © 2020 - Martín Abente Lahaye</property>
+    <property name="copyright">Copyright © 2020 - Martín Abente Lahaye</property>
     <property name="comments" translatable="yes">Flatseal is a graphical utility to review and modify basic permissions from your Flatpak applications.</property>
     <property name="website">https://github.com/tchx84/flatseal</property>
     <property name="website_label" translatable="yes">Flatseal</property>
```

Comment:
Don't forget about removing this translatable tag as well.

Reply:
Done.